mmdet3d/models/middle_encoders/sparse_encoder.py

```diff
@@ -30,7 +30,8 @@ def __init__(self,
                  encoder_channels=((16, ), (32, 32, 32), (64, 64, 64), (64, 64,
                                                                         64)),
                  encoder_paddings=((1, ), (1, 1, 1), (1, 1, 1), ((0, 1, 1), 1,
-                                                                 1))):
+                                                                 1)),
+                 block_type='submblock'):
```

Comment:
1. Update docstrings after line 20 for `block_type`
2. `assert block_type in ['conv_module', 'basicblock']` after line 35